server/config.ts

```diff
@@ -123,11 +123,22 @@ export default {
       path: get('FEATURE_FLAG_PATH', 'feature-flags'),
       filename: get('FEATURE_FLAG_PATH_FILENAME', 'flags.json'),
     },
+    config: {
+      enabled: get('CONFIG_ENABLED', 'true') === 'true',
+      region: get('CONFIG_AWS_REGION', 'eu-west-2'),
+      bucketName: get('CONFIG_BUCKET', 'hmpps-resettlement-passport-ui-config'),
+      path: get('CONFIG_PATH', 'config'),
+      filename: get('CONFIG_PATH_FILENAME', 'config.json'),
+    },
   },
   local: {
     featureFlag: {
       enabled: get('LOCAL_FEATURE_FLAG_ENABLED', 'false') === 'true',
       filename: get('LOCAL_FEATURE_FLAG_PATH_FILENAME', 'localstack/flags.json'),
     },
+    config: {
+      enabled: get('LOCAL_CONFIG_ENABLED', 'false') === 'true',
```

Comment:
Not sure if we want to be able to turn this off since it's an integral part of the design of the feature?